9.5/root/usr/share/container-scripts/postgresql/README.md

```diff
@@ -96,3 +113,19 @@ values stored in the variables and the actual passwords. Whenever a database
 container starts it will reset the passwords to the values stored in the
 environment variables.
 
+
+Troubleshooting
+---------------
+At first the postgres deamon in the container logs to the standard output, so the log is available in the container log. The log can be examined by running:
```

Comment:
Seems to be missing a verb over here. Possibly something along the lines of "At first the postgres daemon writes its logs to the standard output ..."?